tasks/gen/templates/resource/operations/delete_{{underscored_resource}}.cr.ecr

```diff
@@ -0,0 +1,4 @@
+class Delete<%= resource %> < <%= resource %>::DeleteOperation
+  # Read more on deleting records
+  # https://luckyframework.org/guides/database/deleting-records
+end
```

Comment:
Can you add a linebreak after this?

Reply:
Sure 👍 Why is it important to have an empty line at the end of a file?

Reply:
We make sure that all generated code will pass the Ameba tests by default. Even if a project doesn't use Ameba, we still want to make sure they would pass so if someone does add Ameba in, they're not having to fix issues like this.

Reply:
got it, thank you! 🙇